linux_os/guide/system/software/gnome/dconf_use_text_backend/ansible/shared.yml

```diff
@@ -0,0 +1,26 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Force dconf to use the textfiles instead of a binary DB"
+  lineinfile:
+    path: '/etc/dconf/profile/user'
+    regexp: '^service-db:keyfile/user.*'
+    create: yes
```

Comment:
`create` should not be here.

Reply:
Also, should check if file exists before editing else Ansible will throw an error. Here's code used in the templates to prevent that:

https://github.com/ComplianceAsCode/content/pull/3737/files

Reply:
@shawndwells Thank you for your remark, I have double-checked, and discovered that we have have this covered - we have a [test scenario](tests/data/group_system/group_software/group_gnome/rule_dconf_use_text_backend/no_config.fail.sh) that contains a test scenario when the file is absent. I have tested it through and through, and if `create: yes` is missing for the *insert* task, the scenario fails, but when it is missing for the *remove* task, it passes. Therefore, I have modified the PR according to what @yuumasato suggested.